tips-tricks/TroubleshootingandTips.rst

```diff
@@ -667,3 +667,32 @@ creation, you can set the SerDe and properties:
 Notice the double escape required!
 
 .. |image1| image:: ../media/kylo-troubleshooting/2_executesparkjob.png
+
+GetTableData vs ImportSqoop Processor
+========================================================================
+
+Problem
+-------
+
+You need to load data from a structured datastore.
+
+Solution
+--------
+
+There are two major Nifi processors for importing data into Kylo (i.e. into Hadoop): GetTableData and ImportSqoop.
+
+a. **GetTableData** leverages JDBC to pull data from the source into the flowfile within Nifi. This content will then need to be pushed to HDFS (via PutHDFS processor) to the landing area in HDFS.
+
+b. **ImportSqoop** executes a import-sqoop call which pull the content from the source directly to HDFS. 
+
+In general, it is recommended to use the ImportSqoop processor due to performance and minimal IO. Using GetTableData uses the edge node (where Nifi is running) as a middle-man. Also, ImportSqoop runs a MapReduce job which allows you to tune the performance to the data being ingested. For example, if you are loading a reference table, a single mapper will be sufficient while billions of rows would require multiple mappers.
+
+The GetTableData should be used when the data being pulled is small and will be used for further processing. For example, getting the list of tables within a database so additional processing can be done.
```

Comment:
The idea of using GetTableData for tables that need further processing may become confusing for the user. In both cases (GetTableData and ImportSqoop), the tables may need further processing after ingest. Consider providing additional details (or removing this point, whichever is better).